pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -69,6 +70,10 @@ const (
 
 	// ebtables Chain to store dedup rules
 	dedupChain = utilebtables.Chain("KUBE-DEDUP")
+
+	// defaultIPAMDir is the default location for the checkpoint files stored by host-local ipam
+	// https://github.com/containernetworking/cni/tree/master/plugins/ipam/host-local#backends
+	defaultIPAMDir = "/var/lib/cni/networks"
```

Comment:
what if /var/lib is ro on some platform (or do we not care to surface this in a configurable way)?